README.md

```diff
@@ -4,9 +4,9 @@
 ---
 
 OpenThread is an open-source implementation of the [Thread][thread]
-networking protocol. With OpenThread, Nest is making the technology
-used in Nest products more broadly available to accelerate the
-development of products for the connected home.
+networking protocol. Nest is developing OpenThread to make the technology
+used in Nest products more broadly available to developers, which should 
+accelerate the development of products for the connected home.
```

Comment:
This revised sentence now has a different (and misleading) meaning.

The reason for Nest *developing* OpenThread is not "to make the technology used in Nest products more yatta yatta". I think this can be fixed by replacing *"Nest is developing"* with *"Nest has released"*.

I'm not a fan of the use of the word "should" at the end of the sentence—seems too tentative. How about we instead do something like this: Change the *"available to developers, which should accelerate the development of"* to *"available to developers in order to accelerate the development of"*.